shared/src/main/scala/mlscript/NewParser.scala

```diff
@@ -960,8 +884,38 @@ abstract class NewParser(origin: Origin, tokens: Ls[Stroken -> Loc], newDefs: Bo
           exprCont(Quoted(res), prec, allowNewlines)
         case _ :: (IDENT(opStr, true), l0) :: _ if opPrec(opStr)._1 > prec =>
           consume
-          consume
-          opCont(true, opStr, l0, acc, prec, allowNewlines)
+          def insQuotes(term: Term, quoted: Bool): (Term, Bool) = term match {
+            // * If this operator is need to be quoted but has not been
```

Comment:
What does "this operator is need to be quoted" mean? It doesn't even make grammatical sense.

Reply:
I've Updated it.